api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -6335,4 +6335,91 @@ public CohortDefinition getMQ9Den(int flag) {
 
     return cd;
   }
+
+  public CohortDefinition getMQ10Den(boolean adults) {
+
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("revisionEndDate", "revisionEndDate", Date.class));
```

Comment:
Looks like this parameter is NOT applied anywhere in t his cohort definition, it looks redundant for me

Reply:
removed